modules/phase_field/tests/RampIC/tests

```diff
@@ -0,0 +1,8 @@
+[Tests]
+  [./RampIC_test]
```

Comment:
I'm not a fan of tests that are called ```*test```. But let's just leave it that way. I plan on reorganizing the entire phase_field module tests directory anyways... if I end up with too much time on my hands.